test/conformance/kuttl/generate/policy/cornercases/pol-data-create-on-trigger-deletion/01-manifests.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: pol-create-on-trigger-deletion-ns
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-org
+  namespace: pol-create-on-trigger-deletion-ns
+---
+apiVersion: kyverno.io/v1
+kind: Policy
+metadata:
+  name: pol-create-on-trigger-deletion
+  namespace: pol-create-on-trigger-deletion-ns
+spec:
+  rules:
+  - name: default-deny
+    match:
+      any:
+      - resources:
+          kinds:
+          - ConfigMap
+    preconditions:
+      any:
+      - key: "{{ request.operation }}"
+        operator: Equals
+        value: DELETE
+    generate:
+      apiVersion: networking.k8s.io/v1
+      kind: NetworkPolicy
+      name: default-deny
+      namespace: pol-create-on-trigger-deletion-ns
+      synchronize: false
```

Comment:
sync is false here whereas in the clone one it was true. Switch to true, make sure READMEs state sync is enabled.